GETTING_STARTED.md

```diff
@@ -35,7 +35,17 @@ export class PizzaPartyAppModule { }
 
 ## Including core and theme styles:
 This is required to apply all of the core and theme styles to your application.
-See the [theming guide](docs/theming.md) for instructions.
+See the [Theming Guide](docs/theming.md) for instructions.
+
+**From the Theming Guide:**
+
+> If you're using Angular CLI, this is as simple as including one line in your `style.css` file:
+
+> `@import '~@angular/material/core/theming/prebuilt/deeppurple-amber.css';`
+
+> Alternatively, you can just reference the file directly. This would look something like:
+
+> `<link href="node_modules/@angular/material/core/theming/prebuilt/indigo-pink.css" rel="stylesheet">`
```

Comment:
FWIW, although the doc does explicitly say "This is required to apply" and I'm not trying to defend people that don't read the docs:  All of the previous headings before "Including core and themes and styles" are imperative statements ("Install" this.., "Create" that..), however the header for "Including core and theme styles" and all the headers after that ("Additional setup..", "Using Material"..) are passive informational headings (technically imperfect or continuous verbs). If one would just scan the headings, one might come to the incorrect conclusion everything after "Import the Angular Material NgModule" was optional.

I would suggest at least changing the header to "**Include the required core and theme styles**".

Also, if a statement similar to the one below was used, it would let someone accomplish the entirety of the "Getting Started" steps while staying on this page. I think from a usability / user experience standpoint there is a benefit to that.  : 

> "For instance, if using Angular CLI, the required core and theme styles can be included simply by using a pre-built theme by including one line in your `style.css` file:
> `@import '~@angular/material/core/theming/prebuilt/deeppurple-amber.css';`
> See the [theming guide](docs/theming.md) for additional options.
> 

My $.02.